### PR TITLE
Design implementation v2

### PR DESCRIPTION
**Pull request by joshJarr**
Screenshot 2019-06-11 at 17 06 30
Screenshot 2019-06-11 at 17 06 24

### CODE REVIEW

File: src/styles/abstracts/_variables.scss [hunk 1]
```diff
@@ -65,16 +65,16 @@ $font-size-h2: 30px;
 $font-size-h3: 20px;
 $font-size-h4: 12px;
 $font-size-base: 14px;
-$font-size-large: 16px;
+$font-size-large: 18px;
```
Comment (matthewhall) [inline src/styles/abstracts/_variables.scss]: I think this is making `cta-link`s look too large. See the "islands" in the Timeline section.

File: src/styles/components/_section.scss [hunk 2]
```diff
@@ -1,5 +1,5 @@
 .section {
-  overflow-x: hidden;
+  overflow: hidden;
```
Comment (matthewhall) [inline src/styles/components/_section.scss]: What's the reason for changing this?

File: src/styles/abstracts/_variables.scss (hunk 1 above)
Comment (joshJarr) [reply src/styles/abstracts/_variables.scss]: Good point. I've reverted this back after checking the designs.

File: src/styles/components/_section.scss (hunk 2 above)
Comment (joshJarr) [reply src/styles/components/_section.scss]: No reason! removed when testing something and put it back incorrectly.
Though maybe we don't need this rule?

**Pull request merged**